File: src/main/java/no/nav/melosys/eessi/models/sed/nav/Adresse.java

package no.nav.melosys.eessi.models.sed.nav;

import com.fasterxml.jackson.annotation.JsonInclude;
import com.fasterxml.jackson.annotation.JsonInclude.Include;
import lombok.Data;

@SuppressWarnings("unused")
@JsonInclude(Include.NON_NULL)
@Data
public class Adresse {

    private String by;

    private String bygning;

    private String gate;

    private String land;

    private String postnummer;

    private String region;

    private String type;
}

File: melosys-eessi-app/src/main/java/no/nav/melosys/eessi/models/sed/nav/Adresse.kt
package no.nav.melosys.eessi.models.sed.nav

import com.fasterxml.jackson.annotation.JsonInclude

@JsonInclude(JsonInclude.Include.NON_NULL)
data class Adresse(
    var by: String? = null,
    var bygning: String? = null,
    var gate: String? = null,
    var land: String? = null,
    var postnummer: String? = null,
    var region: String? = null,
    var type: String? = null
)
